docs/item.rst

```diff
@@ -0,0 +1,77 @@
+item
+====
+
+.. dfhack-tool::
+    :summary: Perform bulk operations on items based on various properties.
+    :tags: fort productivity items
+
+Filter items in you fort by various properties (e.g., item type, material,
+wear-level, quality, ...), and perform the bulk operations forbid, dump, melt,
+and their inverses. By default, the tool does not consider artifacts. Outputs
+the number of items that matched the filters and were modified.
+
+Usage
+-----
+
+``item [ help | count | [un]forbid | [un]dump | [un]melt ] <filter option>``
+
+Action names should be self explanatory. All actions other than ``help`` and
+``count`` have implicit filters associated with them. For instance, ``item
+forbid --unreachable`` will neither report nor test reachability of items that
+are already forbidden.
+
```

Comment:
needs "Examples" section here, before "Options". it would be a good idea to include a `--description` example for dealing with wool so the warning below has something to point to for how to do it "right"